### ISSUE TITLE
Finetuning setting

### ISSUE THREAD

**Issue by yuellong** [dingdingcai/SC6D-pose#5]
First of all, really nice work and the results are so impressive! I am trying to reproduce the metric on TLESS dataset but I have trouble obtaining the same score as reported in the paper via fine-tuning with real images. I was starting with a learning rate of 1e-4 and gradually annealing it towards 1e-5 after 30 epochs. I also did oversampling of real images to ensure a 50/50 percentage of real / synthetic examples. Any important details that I was missing here? Thanks!

**Comment by dingdingcai** [comment]
How many GPUs are you using for training? The reported results were obtained by training the model on 16 GPUs, which could matter for reproducing the results.

**Comment by yuellong** [comment]
I see. What was your batch size? I was using 128 in my case

**Comment by dingdingcai** [comment]
For a single GPU, the batch_size is set to be 64, so the total batch size should be 64x16=1024

**Comment by yuellong** [comment]
Thanks! Is there any other important changes to hyper parameters in addition to learning rate during finetuning?
